feature-staking-impl/src/main/res/layout/view_alert.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    tools:background="@color/black"
+    tools:parentTag="android.widget.LinearLayout">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingBottom="16dp">
+
+        <TextView
+            android:id="@+id/alertStakeSummaryTitle"
+            style="@style/TextAppearance.Soramitsu.Header5"
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
+            android:layout_weight="1"
+            android:text="@string/staking_alert_title" />
+
+        <View
+            android:id="@+id/view"
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginBottom="8dp"
+            android:background="@color/white_24" />
+
+        <TextView
+            android:id="@+id/alertNoAlertsInfoTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="8dp"
+            android:layout_weight="1"
+            android:text="@string/staking_alert_no_alerts_now" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/alertsRecycler"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            tools:itemCount="3"
+            tools:listitem="@layout/item_alert"
+            android:visibility="gone"/>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Usually having Constaint inside another container is not a good idea since it vanishes benefits of Constraint (flat hierarchy), so it should be avoided, when possible. In this case, it seems like, Contraint can be used as a root for the whole layout

Reply:
I talked to @antonkhvorov about the alerts, and it seems that we don't need this constraint layout there it all